src/thinblock.cpp

```diff
@@ -691,7 +691,7 @@ bool CXThinBlock::process(CNode *pfrom,
     // for a collision.
     int missingCount = 0;
     int unnecessaryCount = 0;
-    bool collision = false;
+    bool collision2 = false;
     std::map<uint64_t, uint256> mapPartialTxHash;
```

Comment:
not sure what this is shadowing...collision is local only and is not getting passed in?